shell/assets/styles/vendor/vue-select.scss

```diff
@@ -103,7 +103,7 @@
   }
 
   &.vs__dropdown-option--highlight {
-    color: var(--dropdown-hover-text);
+    color: var(--dropdown-active-text);
```

Comment:
These names have been created semantically, so if this is for hover, it should stay like this and change the color for it.

Reply:
Fixed